clients/benchmarks/client.cpp

```diff
@@ -36,272 +37,22 @@
 #include "testing_gemm_strided_batched_ex.hpp"
 #endif
 
-namespace po = boost::program_options;
-
-int main(int argc, char* argv[])
+static int run_bench_test(const char* function, char precision, Arguments argus)
 {
-    Arguments argus;
```

Comment:
These diffs are hard to follow. Basically, the running of a performance test has been put into a function by itself, which can either be called from the command-line parser, or from a file reader which reads the Arguments data. strcmp() is used instead of std::string because we want to be able to remove testing_ prefix simply by moving up a pointer, and because it's generally faster.